src/main/java/seedu/tr4cker/storage/JsonSerializableTr4cker.java

```diff
@@ -105,6 +126,8 @@ public Tr4cker toModelType() throws IllegalValueException {
         modulesToModelType(tr4cker);
         tasksToModelType(tr4cker);
         eventsToModelType(tr4cker);
+        modulesToModelType(tr4cker);
```

Comment:
duplicate _modulesToModelType()_, now there are double calls?

Keep the one at the top.